site/static/versions.json

```diff
@@ -1,62 +1,77 @@
 [
     {
         "name": "3.5.4",
-        "version": "3.5.4"
+        "version": "3.5.4",
+        "url": "https://spark.apache.org/docs/3.5.4/api/python/"
     },
     {
         "name": "3.5.3",
-        "version": "3.5.3"
+        "version": "3.5.3",
+        "url": "https://spark.apache.org/docs/3.5.3/api/python/"
```

Comment:
unrelated to this PR, but why isn't there a corresponding path for 3.5.3 in  https://archive.apache.org/dist/spark/docs/ ?